building_sim_plugins/building_gazebo_plugins/src/crowd_simulator.cpp

```diff
@@ -0,0 +1,273 @@
+#include <memory>
+#include <unordered_set>
+#include <regex>
+#include <cstdlib>
+
+#include <ignition/math/Pose3.hh>
+
+#include "crowd_simulator.hpp"
```

Comment:
We normally use double quotations to include header files using relative paths, so this means "In the same folder", so you can move the header files into the `src` instead of `include`. 

Header files that hold public API, to be used by downstream packages, are normally the ones put into the `include` folder, for plugin libraries we are sure that no downstream packages will be using any of the declared public API, hence all the source files for plugins are normally found in `src` and none in `include`, for example `door.cpp`, `lift.cpp`.